webgpu.h

```diff
@@ -492,7 +492,7 @@ typedef enum WGPUCullMode {
 typedef enum WGPUDeviceLostReason {
     WGPUDeviceLostReason_Unknown = 0x00000001,
     WGPUDeviceLostReason_Destroyed = 0x00000002,
-    WGPUDeviceLostReason_InstanceDropped = 0x00000003,
+    WGPUDeviceLostReason_Cancelled = 0x00000003,
```

Comment:
This one is a little awkward. Did this mean the device was lost because the instance was dropped, or just that we _called the device lost callback_ because the instance was dropped?

Reply:
Should we call them all CallbackCancelled or something so the meaning is clearer regardless of context?

Reply:
This is for the device lost callback was called because instance was dropped, i.e. we set device lost callback mode to `ProcessEvents` and instance gets dropped. I'm fine with `CallbackCancelled` instead.

Reply:
Renamed.